### ISSUE TITLE
Feed Me format / compatibility

### ISSUE THREAD

**Issue by darylknight** [craftcms/store-hours#35]
I'm setting up a new site with a list of about a hundred stores, and hoping to use this plugin to manage the opening hours.

I'm planning to use a CSV file for the source which the client will fill in; what's the correct format for getting Feed Me to read store hours and import them correctly to this plugin?

**Comment by mbanta88** [comment]
following

**Comment by jripmeester** [comment]
@darylknight Did you get it to work? I'm having the same task but from JSON.

**Comment by darylknight** [comment]
No, sorry - I ended up having to use Solspace Calendar for this. And the client added all the data themselves instead of us importing.
